Makefile.PL

```diff
@@ -40,8 +40,8 @@ recommends 'DBD::SQLite';
 
 install_share;
 
-install_script 'zonemaster_webbackend.psgi';
-install_script 'zm_wb_daemon';
+install_script 'zonemaster_rpcapi.psgi';
+install_script 'zonemaster_testagent';
 
```

Comment:
Suggest the following even though they are long names:

- zonemaster_backend_rpcapi.psgi
- zonemaster_backend_testagent

Those are not commands that you write on the command line many times.